packages/contracts/src/RelayHub.sol

```diff
@@ -100,6 +105,15 @@ contract RelayHub is IRelayHub, Ownable, ERC165 {
         batchGateway = _batchGateway;
         relayRegistrar = _relayRegistrar;
         setConfiguration(_config);
+        bytes memory randomBytes = Sapphire.randomBytes(32, "");
+        (bytes memory rawPublicKey, bytes memory rawPrivateKey) = Sapphire.generateKeyPair(Sapphire.SigningAlg.Ed25519Oasis, randomBytes);
```

Comment:
In the next runtime upgrade, you'll be able to use `Sapphire.generateCurve25519KeyPair()`, which is the correct thing to use.